addons/hr/hr.py

```diff
@@ -272,7 +272,7 @@ def _message_get_auto_subscribe_fields(self, cr, uid, updated_fields, auto_follo
 
 class hr_department(osv.osv):
     _name = "hr.department"
-    _description = "HR Department"
+    _description = "Department"
```

Comment:
i don't think this one is proper solution..
there is already  create method in hr department.
you should just pass context before super call of create and then call message_post like this way : 
        create_context = dict(context, **mail_create_nolog=True**)
        new_id = super(hr_department, self).create(cr, uid, vals, context=**create_context**)
        self.message_post(cr, uid, new_id, **body=_('Department created')**)